src/components/checkbox/checkbox.html

```diff
@@ -1,27 +1,28 @@
 <fieldset class="bx--fieldset">
-  <legend class="bx--label">Checkbox heading</legend>
+  <legend class="bx--label">Checkbox Heading</legend>
```

Comment:
We use sentence case for all headings and labels, so only the first word should be capitalized